File: 0791-custom-sort-string/0791-custom-sort-string.cpp
class Solution {
public:
    string customSortString(string order, string s) {
        string res = "";
        for (char i : order) {
            size_t c = count(s.begin(), s.end(), i);
            if (s.find(i) != string::npos) {
                res.append(c, i);
            }
        }
        for (char i : s) {
            size_t c = count(s.begin(), s.end(), i);
            if (res.find(i) == string::npos) {
                res.append(c, i);
            }
        }
        return res;
    }
};

File: 0791-custom-sort-string/0791-custom-sort-string.py
class Solution:
    def customSortString(self, order: str, s: str) -> str:
        res=""
        for i in order:
            c=s.count(i)
            if i in s:
                res+=(i*c)
        for i in s:
            c=s.count(i)
            if i not in res:
                res+=(i*c)
        return res